### PR TITLE
configurable keyboard shortcuts

### PR DESCRIPTION
**Pull request by danielweck**
Fixes issue https://github.com/readium/readium-desktop/issues/155

### CODE REVIEW

File: src/main/redux/sagas/keyboard.ts [hunk 1]
```diff
@@ -35,32 +41,65 @@ function* setShortcuts(action: keyboardActions.setShortcuts.TAction) {
...
 function* keyboardShowWatcher() {
     yield takeEvery(keyboardActions.showShortcuts.build, showShortcuts);
 }
+// function* keyboardReloadWatcher() {
+//     yield takeEvery(keyboardActions.reloadShortcuts.build, reloadShortcuts);
+// }
 function* keyboardReloadWatcher() {
-    yield takeEvery(keyboardActions.reloadShortcuts.build, reloadShortcuts);
+    while (true) {
+        const action = yield* takeTyped(keyboardActions.reloadShortcuts.build);
```
Comment (panaC) [inline src/main/redux/sagas/keyboard.ts]: but why ??

you should may be surround `yield takeEvery` with `yield all([takeEvery])` ,
perhaps it's the raison.

File: src/common/redux/reducers/keyboard.ts [hunk 2]
```diff
@@ -22,13 +22,26 @@ export function keyboardReducer(
...
+                    ...{
+                        shortcuts: action.payload.shortcuts,
+                    },
+                }
+            );
         case keyboardActions.showShortcuts.ID:
-            return Object.assign({}, state);
+            return (
+                {
+                    ...state,
```
Comment (panaC) [inline src/common/redux/reducers/keyboard.ts]: it may be useless, right ?

File: src/main/redux/sagas/keyboard.ts (hunk 1 above)
Comment (danielweck) [reply src/main/redux/sagas/keyboard.ts]: There are several issues of this type in this PR. I propose we fix them in a second pass (subsequent PR).

File: src/common/redux/reducers/keyboard.ts (hunk 2 above)
Comment (danielweck) [reply src/common/redux/reducers/keyboard.ts]: There are several issues of this type in this PR. I propose we fix them in a second pass (subsequent PR).

**Comment by danielweck** [comment]
The current set of keyboard shortcuts is described in this Wiki page:
https://github.com/readium/readium-desktop/wiki/Keyboard-shortcuts

**Pull request merged**